src/main/java/automatabuilder/DFA.java

```diff
@@ -58,7 +60,51 @@ public boolean test(String word) throws InvalidWordException {
         
         return end.isFinal();
     }
-    
+
+    @Override
+    public String testFile(File testFile) {
+        int maxLength = 20;
+        StringBuilder result = new StringBuilder();
+        Scanner scanner = null;
+        try {
```

Comment:
I think i'd rather throw an exception to whatever calls the function (might be the GUI in the future)